adapters/ing-psd2-adapter/src/main/java/de/adorsys/xs2a/adapter/service/ing/Headers.java

```diff
@@ -0,0 +1,21 @@
+package de.adorsys.xs2a.adapter.service.ing;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+class Headers {
+    private Map<String, String> headers;
+
+    Headers(Map<String, String> headers) {
+        this.headers = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
+        this.headers.putAll(headers);
+    }
+
+    public String getAccessToken() {
+        String authorization = headers.get("Authorization");
+        if (authorization == null || !authorization.startsWith("Bearer ")) {
+            throw new IllegalArgumentException(authorization);
+        }
+        return authorization.substring(7);
```

Comment:
Could you extract this `7` into some variable with a meaningful name or leave a comment above the `authorization.substring(7);` piece of code, please? As this statement is not so clear for future generations. Thanks!

Reply:
Please see the latest changes.

Reply:
https://github.com/adorsys/xs2a-adapter/compare/772a65914155f59fb525faa6ef37d04dd33cafa3..18242af31e8f52c59d2411f3be409b676bd42161